### PR TITLE
Add/Fix multiple airplane images

### PR DESCRIPTION
**Pull request by Lash12**
Quite new to creating PRs so please let me know if I should change things moving forward. If images should be different please let me know as well!

### CODE REVIEW

File: airline-web/public/images/flags/MM.png
Comment (Lash12) [inline airline-web/public/images/flags/MM.png:1]: Separated this commit into a different PR for flag updates.

**Comment by Lash12** [comment]
@spencerm updated to latest commits to master. This shouldn't cause any issues to merge but if you could take a look I'd appreciate it!

**Comment by spencerm** [comment]
> @spencerm updated to latest commits to master. This shouldn't cause any issues to merge but if you could take a look I'd appreciate it!

Sorry for the delay!! And no I appreciate these PRs! Delay is I want to get it into my local and make sure they actually load. I might need to add one attribute to the Model definitions

**Comment by Lash12** [comment]
> > @spencerm updated to latest commits to master. This shouldn't cause any issues to merge but if you could take a look I'd appreciate it!
> 
> Sorry for the delay!! And no I appreciate these PRs! Delay is I want to get it into my local and make sure they actually load. I might need to add one attribute to the Model definitions

No worries at all, not trying to hound or anything! Also just realized this branch had changes to the sql config that I meant to make to a local testing branch. I reverted those commits so this should just affect the airplane images again.

**Comment by spencerm** [comment]
@Lash12 Merging this, but some of the images won't appear until the planes are updated (which I'll to later)

**Pull request merged**